### ISSUE TITLE
Why pyrogram?

### ISSUE THREAD

**Issue by drizzt** [tropicoo/yt-dlp-bot#259]
Hi,
why are you using pyrogram that is not maintained since a year and it doesn't support catch up (aka you loose messages when the bot is offline)?

**Comment by tropicoo** [comment]
Hi,
Eventually, I will switch to pyrogram fork or Telethon since I need an MTProto client because Bot API allows uploading up to 50MB files while MTProto is up to 2GB.

**Comment by drizzt** [comment]
yes, I know... well, I currently use a telegram-bot-api locally in order to use 2GB files, but telethon is a good choice
